libclambcc/ClamBCVerifier/ClamBCVerifier.cpp

```diff
@@ -103,7 +103,7 @@ static cl::opt<bool>
     StopOnFirstError("clambc-stopfirst", cl::init(false),
                      cl::desc("Stop on first error in the verifier"));
 #else
-static bool StopOnFirstError = true;
+//static bool StopOnFirstError = true;
```

Comment:
Did you mean to commit this?

Reply:
It was an option in the first run version of the compiler, but I think I am going to remove it entirely.  I left it in to remember to ask if we need it.